File: mobile/ios/Voice/ServerConfigIos.m
#import <React/RCTBridgeModule.h>
#import "Voice-Swift.h"

@interface ServerConfigIosModule : NSObject <RCTBridgeModule>
@end

@implementation ServerConfigIosModule

RCT_EXPORT_MODULE(ServerConfigIos);

+ (BOOL)requiresMainQueueSetup {
  return NO;
}

- (NSDictionary*)constantsToExport {
  return @{
    @"devHost" : [ServerConfigIos devHost],
    @"prodHost" : [ServerConfigIos prodHost],
  };
}

@end

File: mobile/ios/Voice/ServerConfigIos.swift
import Foundation

@objc(ServerConfigIos)
final class ServerConfigIos: NSObject {
  @objc static let devHost = "admins-MacBook-Pro.local"
  @objc static let prodHost = "voice.k.vu"

  static var baseURL: String {
    #if DEBUG
      return "https://\(devHost):3003"
    #else
      return "https://\(prodHost)"
    #endif
  }
}
